sqlobject/src/main/java/org/jdbi/v3/sqlobject/statement/internal/SqlBatchHandler.java

```diff
@@ -330,6 +327,7 @@ public boolean hasNext() {
 
             @Override
             public Object[] next() {
+                final Object[] sharedArg = new Object[args.length];
```

Comment:
The argument array is no longer shared, so how about just `args` or `argArray`

Reply:
Updated